.github/workflows/scale-test-100-gce.yaml

```diff
@@ -28,8 +31,6 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  # renovate: datasource=github-releases depName=kubernetes/kops
```

Comment:
Can we keep this variable here but set it to the alpha release that we need? It would be nice to be able to keep renovate.

Reply:
I'm not sure about it. Let's say that we will be using some new flags that we will be adding to scale-test-actions that require a specific version of kops like right now. Imho it makes more sense to specify kops version that is compatible with scale-test-actions in scale-test-actions repo.
We could potentially add renovate to scale-test-actions and here only update scale-test-actions - I think it would make more sense than having them being updated separately as they depend on each other.

Reply:
I see where you are coming from, that makes sense. I'll work on adding renovate to the scale-test-actions repository.

Reply:
https://github.com/cilium/scale-tests-action/pull/5

Reply:
Thanks!